django/db/backends/base/schema.py

```diff
@@ -798,7 +799,9 @@ def _create_index_name(self, model, column_names, suffix=""):
             )
         # Else generate the name for the index using a different algorithm
         table_name = model._meta.db_table.replace('"', '').replace('.', '_')
-        index_unique_name = '_%x' % abs(hash((table_name, ','.join(column_names))))
```

Comment:
I agree with @timgraham here -- The patch in the current form has serious backwards incompatibility problems for cases which are currently working fine (py2 without hash randomization, ie our biggest userbase).